test/unitest_solution_gen.cpp

```diff
@@ -18,23 +18,44 @@ namespace mmh3 {
 
 int main( int argc, const char** argv )
 {
-	FILE* f   = fopen( "test/pg1260.txt", "rb" );
-	FILE* out1 = fopen( "test/solution_hash32_seed0.txt", "wb" );
-	FILE* out2 = fopen( "test/solution_hash32_seed1234ABCD.txt", "wb" );
-	char buffer[2048];
+    FILE* f    = fopen( "test/pg1260.txt", "rb" );
+    FILE* out1 = fopen( "test/solution_hash32_seed0.txt", "wb" );
+    FILE* out2 = fopen( "test/solution_hash32_seed1234ABCD.txt", "wb" );
+    FILE* out3 = fopen( "test/solution_hash128_x86_seed0.txt", "wb" );
```

Comment:
x86 and x64 implementations give different hash values by design.
You can verify this using mmh3, the c++ implementation.